ml-fundamentals/ml-problem-formulation-envisioning.md

```diff
@@ -0,0 +1,115 @@
+
+A better objective, in this case, would be "to drive additional sales by presenting the customer with items that they *would not otherwise have purchased without the recommendation*". Here, the inputs that we can control are the choice of items that are presented to each customer, and the order in which they are displayed; considering factors such as how frequently these should change, seasonality, etc.
+
+The data required to evaluate a potential solution in this case would be which recommendations resulted in new sales, and an estimation of a customer's likeliness to purchase a specific item without a recommendation. Note that, whilst this data could also be used to build a recommendation engine, it is unlikely that this data will be available before a recommendation system has been implemented, so it is likely that we will have to use an alternate data source to build the model.
+
+We can get an initial idea of how to approach a solution to this problem by considering how it would be solved by a subject matter expert. Thinking of how a personal stylist may provide a recommendation, they are likely to recommend items based on one or more of the following:
+
+* generally popular items
+* items similar to those liked/purchased by the customer
+* items that were liked/purchased by similar customers
+* items which are complementary to those owned by the customer
+
+Whilst this list is by no means exhaustive, it provides a good indication of the data that is likely to be useful to us:
+
+* item sales data
+* customer purchase histories
+* customer demographics
+* item descriptions and tags
+* previous outfits, or sets, which have been curated by the stylist
+
+We would then be able to use this data to explore:
+
+* a method of measuring similarity between items
+* a method of measuring similarity between customers
+* a method of measuring how complementary items are relative to one another
+
+which can be used to create and rank recommendations. Depending on the project scope, and available data, one or more of these areas could be selected to create hypotheses to be explored by the data science team. Some examples of such hypothesis statements could be:
+
+* From the descriptions of each item, we can determine a measure of similarity between different items to a degree of accuracy which is specified by a stylist.
+* Based on the behaviour of customers with similar purchasing histories, we are able to predict certain items that a customer is likely to purchase; with a certainty which is greater than random choice.
```

Comment:
```suggestion
* Based on the behavior of customers with similar purchasing histories, we are able to predict certain items that a customer is likely to purchase; with a certainty which is greater than random choice.
```

Reply:
Are we going U.S or UK? ;)

Reply:
Fair point :) I go through most PRs and have tried to standardize on US English for a cohesive feeling - but having said that. I won’t reject if you refuse to change this